app/components/CustomCard/tests/index.test.js

```diff
@@ -0,0 +1,22 @@
+/**
+ *
+ * Tests for CustomCard
+ *
+ */
+
+import React from 'react';
+// import { fireEvent } from '@testing-library/dom'
+import { renderWithIntl } from '@utils/testUtils';
+import CustomCard from '../index';
+
+describe('<CustomCard />', () => {
```

Comment:
add more tests

Reply:
add tests to ensure that when name, fullname and stargazersCount is sent the appropriate things are rendered